filament/src/Renderer.cpp

```diff
@@ -731,7 +731,7 @@ FrameGraphId<FrameGraphTexture> FRenderer::colorPass(FrameGraph& fg, const char*
                             // MS, no need to allocate the depth buffer with MS, if the RT is MS,
                             // the tile depth buffer will be MS, but it'll be resolved to single
                             // sample automatically -- which is what we want.
-                            .samples = colorBufferDesc.samples,
+                            .samples = colorBufferDesc.samples == 0 ? uint8_t(1) : colorBufferDesc.samples,
```

Comment:
Could we make sure the sample count is never 0 in the description instead? It seems error prone if we have to make this check everywhere we read the sample count.

Reply:
Yeah I think we should fix that in the ResourceAllocator instead.